pkg/k8s/apis/cilium.io/v1alpha1/podinfo_types.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2023 Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	ciliumio "github.com/cilium/tetragon/pkg/k8s/apis/cilium.io"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// PodInfo (PI) is the custom resource that stores pod related information
+
+	// PIPluralName is the plural name of Tetragon Pod Info
+	PIPluralName = "podsinfo"
```

Comment:
```suggestion
	PIPluralName = "podinfo"
```
i think it's ok to use `podinfo` for plural

Reply:
ok then singular and plural both will be named podinfo